core/src/main/java/io/questdb/cutlass/line/tcp/TableUpdateDetails.java

```diff
@@ -196,63 +194,61 @@ TableWriter getWriter() {
         return writer;
     }
 
-    void handleRowAppended() {
-        if (commitIfMaxUncommittedRowsCountReached(writer)) {
-            lastCommitMillis = millisecondClock.getTicks();
-        }
-    }
-
-    private boolean commitIfMaxUncommittedRowsCountReached(TableWriter writer) {
+    void commitIfMaxUncommittedRowsCountReached() {
         final long rowsSinceCommit = writer.getUncommittedRowCount();
         if (rowsSinceCommit < writer.getMetadata().getMaxUncommittedRows()) {
             if ((rowsSinceCommit & writerTickRowsCountMod) == 0) {
                 // Tick without commit. Some tick commands may force writer to commit though.
                 writer.tick(false);
             }
-            return false;
+            return;
         }
         LOG.debug().$("max-uncommitted-rows commit with lag [").$(tableNameUtf16).I$();
-        writer.commitWithLag(engine.getConfiguration().getCommitMode());
+        nextCommitTime = millisecondClock.getTicks() + writer.getCommitInterval();
+        writer.commitWithLag();
         // Tick after commit.
         writer.tick(false);
-        return true;
     }
 
-    boolean commitIfTimeoutElapsed(long millis) {
-        if (millis - lastMeasurementMillis < noAppendInterval || millis - lastCommitMillis < commitTimeout) {
-            return false;
+    long checkIfTableNeedsCommit(long millis) {
```

Comment:
nit: could we rename `millis` to `wallClockMillis` in this method and everywhere its used?

Reply:
done